lib/components/form/advanced-settings-panel.tsx

```diff
@@ -169,6 +169,7 @@ const AdvancedSettingsPanel = ({
   )
 
   const processedModeSettings = processSettings(modeSettingDefinitions)
+
```

Comment:
can you remove this new line? (it feels unimportant but it keeps the git blame cleaner)